Controller/PreSymfony6AbstractFOSRestController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Controller;
+
+use FOS\RestBundle\View\ViewHandlerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+/**
+ * @internal
+ */
+abstract class PreSymfony6AbstractFOSRestController extends AbstractController
+{
+    use ControllerTrait;
+
+    /**
+     * {@inheritdoc}
```

Comment:
```suggestion
     * @return array
```

Should we add this return type hint in order to silence the deprecation notice in the symfony 5.4 pipeline?

```
2x: Method "Symfony\Contracts\Service\ServiceSubscriberInterface::getSubscribedServices()" might add "array" as a native return type declaration in the future. Do the same in implementation "FOS\RestBundle\Controller\PreSymfony6AbstractFOSRestController" now to avoid errors or add an explicit @return annotation to suppress this message.
    2x in AllowedMethodsTest::testAllowHeader from FOS\RestBundle\Tests\Functional
```